testsupport/space.go

```diff
@@ -44,9 +44,9 @@ func NewSpace(awaitilities wait.Awaitilities, opts ...SpaceOption) *toolchainv1a
 
 type SpaceOption func(*toolchainv1alpha1.Space)
 
-func WithTargetCluster(memberCluster *wait.MemberAwaitility) SpaceOption {
+func WithTargetCluster(clusterName string) SpaceOption {
 	return func(s *toolchainv1alpha1.Space) {
-		s.Spec.TargetCluster = memberCluster.ClusterName
+		s.Spec.TargetCluster = clusterName
```

Comment:
just out of curiosity - what is the benefit of this change?

Reply:
it's more explicit than having no parameter and having to guess what member cluster is used at the caller level.

Reply:
But that was explicit before as well - the original code took the entire instance of the MemberAwaitility `memberCluster *wait.MemberAwaitility` and used the name of it, but you changed it to passing the name of the member clustre as a string. The original code was clear with what the function accepted - an instance of `MemberAwaitility`, with passing a string you can easily make a typo in the name.

never mind - it's just a detail - I was just curious if there was any real reason for changing that, but it can stay as it is...

Reply:
this `WithTargetCluster(string)` func is also used in the host-operator repository where there is no such `*wait.MemberAwaitility`. Here, I used `memberCluster.ClusterName` as an arg to call this func, so there should never be any typo.